kube_scheduler/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "display_name": "Kube_scheduler",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "kube_scheduler",
+  "metric_prefix": "kube_scheduler.",
+  "metric_to_check": "",
```

Comment:
You should add a value for `metric_to_check` since metrics are emitted: https://github.com/DataDog/integrations-core/blob/master/docs/dev/new_check_howto.md#manifestjson